wcomponents-core/src/main/java/com/github/bordertech/wcomponents/template/HandlebarsRendererImpl.java

```diff
@@ -54,6 +56,16 @@
 	public static final String ESCAPING_STRATEGY = "ESCAPING_STRATEGY";
 
 	/**
+	 * Perform theme i18n on the server, defaults is "true" (theme i18n bundle must be on the classpath).
+	 */
+	public static final String THEME_I18N = "THEME_I18N";
+
+	/**
+	 * The basename of the theme i18n resource bundle. (TODO should this be a configuration property?)
+	 */
+	private static final String THEME_I18N_RESOURCE_BUNDLE_BASE_NAME = "com/github/bordertech/wcomponents/theme-messages";
```

Comment:
Noticed you half setup the parameter name for this in the `ConfigurationProperties`. Might as well set it up with this PR.